src/transformers/models/aimv2/convert_aimv2_to_hf.py

```diff
@@ -0,0 +1,225 @@
+# coding=utf-8
```

Comment:
Please refactor it to follow `mllama` model conversion format. It should be a KEY_MAPPING dict instead of `create_rename_keys`